docsrc/conf.py

```diff
@@ -527,8 +527,8 @@
 # XXX handlers here, except that these urls require expanding the version string
 # XXX twice, and the sphinx extlinks thingy currently only supports a single %s
 extlinks = {
-        'draft':('https://tools.ietf.org/html/%s', ''),
-        'issue':('https://github.com/cyrusimap/cyrus-imapd/issues/%s', 'Issue #'),
+        'draft':('https://tools.ietf.org/html/%s', '%s'),
+        'issue':('https://github.com/cyrusimap/cyrus-imapd/issues/%s', 'Issue #%s'),
```

Comment:
We will need to make this change eventually to support Sphinx 6, but we can't yet, because it breaks backward compatibility with Sphinx 3, which is what's used to build the cyrusimap.org website.  More discussion here: https://github.com/cyrusimap/cyrus-imapd/issues/4491

In the meantime, we support up to Sphinx 5, and ignore this warning

Reply:
Okay, may I drop the commit?